MTM-61893 Add description for button which opens documentation for Custom properties (#2702)
Cumulocity-IoT/c8y-docs: content/enterprise-tenant/managing-tenants-bundle/custom-properties.md

weight: 30
title: Custom properties
layout: redirect
helpcontent:
  - label: custom-properties
    title: Custom properties

>     content: "The Custom properties tab allows you to edit and set only values of tenant custom properties defined earlier by user in the properties library. Fields requirements, types and regular expressions can be changed in the properties library."

Reviewer: ```suggestion
    content: "The **Custom properties** tab allows you to edit and set values for tenant custom properties which have been defined in the properties library. Fields requirements, types and regular expressions can be changed in the properties library."
```
